### PR TITLE
arcane battery

### PR DESCRIPTION
**Pull request by Criticalmage113**
Items needed for Arcane Battery

### CODE REVIEW

**Comment by Gearp0wer** [comment]
side note, I feel like more than star and I should be able to approve these but idk, thats just me

File: Dev Pack/assets/minecraft/models/item/heart_of_the_sea.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
     {"predicate": { "custom_model_data": 11002}, "model": "valence:weapon/charm/lunar_charm"},
     {"predicate": { "custom_model_data": 11003}, "model": "valence:misc/citadelsignet"},
     {"predicate": { "custom_model_data": 11004}, "model": "valence:misc/pocket_conduit"},
-    {"predicate": { "custom_model_data": 11005}, "model": "valence:misc/inquisitor/nullstone_amp"}
+    {"predicate": { "custom_model_data": 11005}, "model": "valence:misc/inquisitor/nullstone_amp"},
+	{"predicate": { "custom_model_data": 11006}, "model": "valence:weapon/charm/stag_charm"},
+	{"predicate": { "custom_model_data": 11007}, "model": "valence:misc/the_godseed"}
```
Comment (Starmism) [inline Dev Pack/assets/minecraft/models/item/heart_of_the_sea.json:14]: These should also be unindented

File: Dev Pack/assets/minecraft/models/item/book.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     },
     "overrides": [
       {"predicate": {"custom_model_data": 11001}, "model": "valence:weapon/magic/arcane_spellbook"},
+	  {"predicate": {"custom_model_data": 11002}, "model": "valence:weapon/magic/maldores_spellbook"},
```
Comment (Starmism) [inline Dev Pack/assets/minecraft/models/item/book.json:8]: This should be unindented and should have an empty line after it to signify the jump from 11000 to 14000

File: Dev Pack/assets/minecraft/models/item/stick.json [hunk 3]
```diff
@@ -60,6 +60,7 @@
 
 
       {"predicate": {"custom_model_data": 11049}, "model": "valence:resources/raw/bundle_scrapwood2"},
+	  {"predicate": {"custom_model_data": 11050}, "model": "valence:weapon/magic/maldore_wand2_1"},
```
Comment (Starmism) [inline Dev Pack/assets/minecraft/models/item/stick.json:63]: Should be unindented

**Review by Starmism — CHANGES_REQUESTED** [review]

**Review by Starmism — APPROVED** [review]

**Pull request merged**